app/uk/gov/hmrc/payetaxcalculatorfrontend/model/UserTaxCode.scala

```diff
@@ -31,34 +35,36 @@ object UserTaxCode extends TaxCalculatorHelper {
 
   implicit val format = Json.format[UserTaxCode]
 
-  val defaultTaxCode = "1150L"
-  val hasTaxCode = "hasTaxCode"
-  val taxCode = "taxCode"
+  val DEFAULT_TAX_CODE = "1150L"
```

Comment:
I know this is JAVA style, but this way we know exactly those variables were hardcoded ;).